GoogleUtilities/AppDelegateSwizzler/Public/GoogleUtilities/GULSceneDelegateSwizzler.h

```diff
@@ -17,18 +17,19 @@
 #import <Foundation/Foundation.h>
 #import <TargetConditionals.h>
 
-#if !TARGET_OS_OSX
+#if __has_include(<UIKit/UIKit.h>)
 #import <UIKit/UIKit.h>
-#endif  // !TARGET_OS_OSX
+#endif
 
-#if ((TARGET_OS_IOS || TARGET_OS_TV) && (__IPHONE_OS_VERSION_MAX_ALLOWED >= 130000))
+#if TARGET_OS_IOS || TARGET_OS_TV
 #define UISCENE_SUPPORTED 1
 #endif
 
 NS_ASSUME_NONNULL_BEGIN
 
 typedef NSString *const GULSceneDelegateInterceptorID;
 
+// TODO(v8): Update API availability so this class is only available on iOS/tvOS 13.0+
```

Comment:
why?

Reply:
Deleting. I can't remember what I was thinking at the time.